lib/v2/huangz/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'huangz.me': {
+        _name: 'huangz/blog',
```

Comment:
`_name` should match H2 heading of the docs
```suggestion
        _name: '黄健宏博客',
```